libcontainer/intelrdt/intelrdt.go

```diff
@@ -532,14 +518,18 @@ func IsMBAScEnabled() bool {
 }
 
 // Get the 'container_id' path in Intel RDT "resource control" filesystem
```

Comment:
Perhaps container_id -> ClosID ?

Reply:
Good catch, thanks. I addressed this in a separate patch, modifying some other code comments as well. I used the term *clos group*